videointelligence/video_quickstart/main.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2017 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START video_quickstart]
+
+// Sample video_quickstart uses the Google Cloud Video Intelligence API to label a video.
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/golang/protobuf/ptypes"
+
+	// Imports the Google Cloud Video Intelligence API client package.
+	video "cloud.google.com/go/videointelligence/apiv1"
+	"golang.org/x/net/context"
+	videopb "google.golang.org/genproto/googleapis/cloud/videointelligence/v1"
+)
+
+func main() {
+	ctx := context.Background()
+
+	// Creates a client.
+	client, err := video.NewClient(ctx)
+	if err != nil {
+		log.Fatalf("Failed to create client: %v", err)
+	}
+
+	op, err := client.AnnotateVideo(ctx, &videopb.AnnotateVideoRequest{
+		InputUri: "gs://demomaker/cat.mp4",
```

Comment:
I created a `cloud-storage-samples` bucket that might make sense to use for samples like this.

Reply:
```
AccessDeniedException: 403 cbro@google.com does not have storage.objects.list access to cloud-storage-samples.
```

Reply:
Oops, I forgot to make it public. I've made you owner of the `storage-samples` project that owns that bucket. I'd make it public myself but am WFH without a corp cert right now.

Reply:
Ah, I can't make you an owner with the app, either. @frankyn might be able to help.

Reply:
You are an admin now @broady!

Reply:
Merging now without this change. Please file a bug if this URI isn't appropriate for the quickstart.